### PR TITLE
Changed modal content and data structure

### PR DESCRIPTION
**Pull request by ManuelOviedo**
#3 
Hope this helps.
I added nodemon for avoiding restart server every time it's contents are changed

### CODE REVIEW

**Comment by ManuelOviedo** [comment]
Done

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A weather report app",
   "main": "server/app.js",
   "scripts": {
-    "start": "node server/app.js",
+    "start": "nodemon -x node server/app.js",
```
Comment (ManuelOviedo) [inline package.json]:
```suggestion
    "start": "node server/app.js",
```

**Comment by sudipt1999** [comment]
Looks Great! thanks for the contribution!

**Comment by ManuelOviedo** [comment]
You are welcome!
Anytime i can help, just let me know!

**Pull request merged**